source/utilities.cc

```diff
@@ -2363,6 +2364,84 @@ namespace aspect
         }
     }
 
+    Operator::Operator()
+      :
+      op(uninitialized)
+    {}
+
+
+    Operator::Operator(const operation _op)
+      :
+      op(_op)
+    {}
+
+
+    double
+    Operator::operator() (double x, double y) const
+    {
+      switch (op)
+        {
+          case Utilities::Operator::add:
+          {
+            return x + y;
+            break;
+          }
+          case Utilities::Operator::subtract:
+          {
+            return x - y;
+            break;
+          }
+          case Utilities::Operator::minimum:
+          {
+            return std::min(x,y);
+            break;
+          }
+          case Utilities::Operator::maximum:
+          {
+            return std::max(x,y);
+            break;
+          }
+          default:
+          {
+            Assert ( false, ExcMessage ("ASPECT's operator class only supports the following operators: "
+                                        "add, subtract, minimum and maximum. Please check your parameter file.") );
```

Comment:
I can get here if somebody used the default constructor, and I can not remove the default constructor, because I currently use it in `create_model_operator_list` to resize a std::vector. I could work around this, if you want to make sure this never happens.